tests/tests.rs

```diff
@@ -72,6 +72,7 @@ fn compile_test() {
     let () = rustc_build_sysroot::SysrootBuilder::new(&directory, target)
         .build_mode(rustc_build_sysroot::BuildMode::Build)
         .sysroot_config(rustc_build_sysroot::SysrootConfig::NoStd)
+        .rustflag("-Cdebuginfo=2")
```

Comment:
can you add a comment here to explain this? can we rely on the sysroot shipped with rustup to have DI?

Reply:
You'll probably see it when you build some BPF crate using `-Z build-std=core`; if it builds without DI, we might have a problem.

The fact that you're using `cargo test` doesn't mean anything because the tests are building the sysroot (right here).

Reply:
`-Zbuild-std=core` will build core, so it'll generate DI if DI `-C debuginfo=2` is passed